region/maridia/inner-pink/Halfie Climb Room.json

```diff
@@ -3333,9 +3385,25 @@
       ]
     },
     {
-      "id": 116,
       "link": [4, 1],
       "name": "Grapple Teleport",
+      "entranceCondition": {
```

Comment:
Seems pointless if there is no doorLockBypass to this strat.
Do `comeInWithGrappleTeleport`s have an implicit "just don't hold the button and you stay at the door"?